datafusion/datasource-parquet/src/source.rs

```diff
@@ -490,6 +497,15 @@ impl ParquetSource {
             )),
         }
     }
+
+    pub fn with_reverse_scan_inexact(mut self, reverse_scan_inexact: bool) -> Self {
+        self.reverse_scan_inexact = reverse_scan_inexact;
+        self
+    }
+
+    pub fn reverse_scan_inexact(&self) -> bool {
+        self.reverse_scan_inexact
+    }
```

Comment:
not a huge fan of tying these to the public API.

I would follow the pattern that pushdown of fitlers / projection have and propagate the full `try_pushdown_sort` into the `FileSource` layer.

Or if these are just to be used from tests make them `#[cfg(test)]` or `pub(crate)`.